src/tasks.py

```diff
@@ -21,32 +38,237 @@ def on_failure(self, _exc, _task_id, _args, _kwargs, _einfo):
 
 
 # noinspection PyUnnecessaryBackslash
-def create_model_for_area(selected_polygon_wkt: str) -> result.GroupResult:
-    """Creates a model for the area using series of chained (sequential) and grouped (parallel) sub-tasks"""
-    return (initialise_db_with_region_geometries.si() |
-            download_lidar_data.si(selected_polygon_wkt) |
+def create_model_for_area(selected_polygon_wkt: str, scenario_options: dict) -> result.GroupResult:
+    """
+    Creates a model for the area using series of chained (sequential) and grouped (parallel) sub-tasks.
+
```

Comment:
This should reflect what is actually hapening (chained)
Add a comment below to show that grouped tasks could be added in the future. Add issue for this